ERCS/erc-7866.md

```diff
@@ -0,0 +1,346 @@
+    "github": "https://github.com/anistark"
+  },
+  "default_avatar_visibility": "public", 
+  "dapp_avatars": {
+    "0xDAppAddress1": {
+      "avatar": "ipfs://<CID>",
+      "visibility": "private"
+    },
+    "0xDAppAddress2": {
+      "avatar": "ipfs://<CID>",
+      "visibility": "public"
+    }
+  }
+}
+```
+
+#### Access Control
+
+1. **Default Avatar Visibility**: Users can set their default avatar visibility as public or private.
+2. **dApp-Specific Avatar Visibility**: Each dApp-specific avatar can also have its visibility set to public or private.
+
+**Visibility Logic**:
+- If an avatar is public, it is retrievable by any external caller.
+- If an avatar is private, only the user can retrieve it. Other callers will get an encoded response alongwith error message.
+
+### dApp-Specific Avatar Customisation
+- Users can assign dApp-specific avatars or metadata.
+- If no dApp-specific customisation exists, the default avatar applies.
+
+## Rationale
+
+The design of the Decentralised Profile Standard was guided by the need for a unified, interoperable user profile system that can operate seamlessly across all blockchain networks. Current solutions, such as ENS profiles or Gravatar, either lack cross-chain functionality, are centralised, or do not allow users to customise profiles for specific dApps. This standard addresses these shortcomings while ensuring simplicity, security, and scalability. 
+
+### Design Decisions
+
+1. **Decentralised Identifiers (DIDs)**
+   - Using the `did:<chain>:<address>` format provides a globally unique identifier for profiles. This aligns with the decentralised identity movement and ensures compatibility with broader DID frameworks.
+
+2. **Decentralised Profile**
+   - The profile format (`username@networkslug.soul`) makes profiles human-readable and chain-specific while maintaining a universal structure. The `.soul` suffix clearly identifies profiles compliant with this standard.
+
+3. **dApp-Specific Avatars**
+   - Allowing users to assign dApp-specific avatars caters to personalisation and enhances the user experience. It supports scenarios where users may want different representations or metadata for different applications.
+
+4. **Soul Bound Tokens (SBTs)**
+   - Leveraging SBTs ensures that profiles are non-transferable, reinforcing the concept of identity ownership. SBTs prevent profiles from being sold or hijacked, making them ideal for reputation-based systems.
+
+5. **Registry and Resolver Architecture**
+   - This architecture was chosen for its extensibility and proven track record, as seen in ENS. It separates the management of profile identifiers (Registry) from the resolution of metadata (Resolver), making upgrades and integrations straightforward.
+
+6. **Compatibility with Existing Standards**
+   - The profile standard integrates with `ERC165` for interface detection and can complement ENS or other naming systems, fostering interoperability rather than competition.
```

Comment:
```suggestion
   - The profile standard integrates with [ERC-165](./eip-165.md) for interface detection and can complement ENS or other naming systems, fostering interoperability rather than competition.
```